xamarin/Xamarin.Forms
# [iOS] If SwipeView is disabled, tap gesture events are not firing

### Description of Change ###

Disabling the SwipeView should disable the swipe gesture, but not the interaction with the content. If you want to disable the interaction with the content, must disable the Content.
With the changes in this PR, the behavior must be the same on Android, iOS.

### Issues Resolved ### 

- fixes #11969
- fixes #11968

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix11969](https://user-images.githubusercontent.com/6755973/99952741-6d6f3300-2d80-11eb-81a8-000488dc6bc3.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11969. Disabling the SwipeView should disable the swipe gesture, but not the interaction with the content. If you want to disable the interaction with the content, must disable the Content.

### PR Checklist ###
